docs/usage/getting-started/use-cases.md

```diff
@@ -0,0 +1,135 @@
+Automerge is particularly useful for internal dependencies when it's best to use the approach of "if it passes tests then merge it".
+
+## Advanced Configuration Use
+
+The below capabilities are common across the above use cases.
+
+### Batched Updates
+
+Although Renovate defaults to separating each dependency update into its own PR, some users prefer to batch or "group" updates together.
+For example, group all patch updates into one PR or even perhaps all non-major updates together (patches and minor updates).
+
+Renovate supports this capability using the `groupName` configuration as part of `packageRules`.
+
+### Scheduled Updates
+
+Some users prefer to limit which hours of the day or week during which Renovate will raise updates.
+This may be to reduce "noise" during working hours, but also to reduce the chance of CI contention at times when developers are more likely to be waiting on tests to finish.
+
+Renovate allows users to define time ranges during which to create updates using the `schedule` field.
+
+### On-demand Updates
+
+Renovate's "Dependency Dashboard" capability is supported on platforms which support dynamic Markdown checkboxes (GitHub, GitLab, and Gitea).
+When enabled, an issue titled "Dependency Dashboard" is created which lists all updates which are pending, in progress, or were previously closed ignored.
+
+Importantly, it also enables the concept of "Dependency Dashboard Approval", meaning that configured PRs won't be raised automatically and will instead only be created once the corresponding checkbox is clicked in the dashboard.
+This can be an improvement in two ways:
+
+- By not raising PRs automatically, it can allow users to request them on-demand at times when they are ready to take action on them, and
+- Offering an alternative to permanently ignoring/disabling certain types of updates, such as major updates
+
+By having this dashboard concept it gives users both visibility and control over updates.
+
+### Configuration Presets
+
+It's pretty common that users will run Renovate on many repositories and want mostly similar config on them all too.
+Renovate supports the concept of configuration "presets" to avoid users needing to duplicate configuration across all such repos.
+
+Configuration presets are JSON configuration files which are committed to repositories and then referenced from others.
+Renovate also includes over 100 built-in presets, including the default recommended `config:base` preset.
+
+The typical workflow for a company is:
+
+- Create a dedicated repository to store the company's default Renovate settings
+- Set that repository as the default `extends` value when onboarding new repositories
+
+This means that repositories get the centralized config by default, while any changes made to the centralized config repository are then propagated out to other repositories immediately.
```

Comment:
We might want to mention that you can override the "global/centralized" config by putting overriding config in the repositories own config file?

Reply:
I think too much detail for here. We should have a dedicated page to the use of presets

Reply:
Ah okay, let's leave this as-is then. 😄